deploy/playbooks/roles/common/tasks/nginx.yml

```diff
@@ -42,19 +42,19 @@
   sudo: true
 
 - name: check for SSL cert
-  stat: path=/etc/ssl/certs/{{ ansible_fqdn }}-bundled.crt
+  stat: path=/etc/ssl/certs/{{ prod_fqdn }}-bundled.crt
   ignore_errors: true
   register: ssl_cert
 
 - name: create self-signed SSL cert
-  command: openssl req -new -nodes -x509 -subj "/C=US/ST=Oregon/L=Portland/O=IT/CN=${app_name}" -days 3650 -keyout /etc/ssl/private/{{ ansible_fqdn }}.key -out /etc/ssl/certs/{{ ansible_fqdn }}-bundled.crt -extensions v3_ca creates=/etc/nginx/ssl/{{ ansible_fqdn }}-bundled.crt
+  command: openssl req -new -nodes -x509 -subj "/C=US/ST=Oregon/L=Portland/O=IT/CN=${app_name}" -days 3650 -keyout /etc/ssl/private/{{ prod_fqdn }}.key -out /etc/ssl/certs/{{ prod_fqdn }}-bundled.crt -extensions v3_ca creates=/etc/nginx/ssl/{{ prod_fqdn }}-bundled.crt
   when: development_server and ssl_cert.stat.exists == false
   sudo: true
   notify: restart nginx
 
 - name: make sure permissions are correct for crt
   file:
-    path: /etc/ssl/certs/{{ ansible_fqdn }}-bundled.crt
+    path: /etc/ssl/certs/{{ prod_fqdn }}-bundled.crt
     mode: 0777
   when: development_server
```

Comment:
development_server is set to false in deploy.yml, is this cert only going to be used with vagrant testing?